### ISSUE TITLE
formDidFailLoading with "Unable to load the form" on iOS 15 device when build generated from Xcode 12.5.1

### ISSUE THREAD

**Issue by Rajkumar2432** [usabilla/usabilla-u4a-ios-swift-sdk#228]
I am installing the build generated from Xcode 12.5.1 on iOS 15 device where Usabilla form does not presents a page. It was working on iOS 14 and it is working on iOS 15 when running from Xcode 13. I am facing this issue only on Xcode 12.5 build on iOS 15 device.

**Comment by xanderwernecke** [comment]
Hi there,

Thank you for reaching out! Please could you provide me with the following and I will forward it onto our Developers:

- Platform: From your message, it's iOS 
- SDK version
- Device & Device OS version
- Steps to reproduce the issue
- FormId / AppId / CampaignId
- Stack trace / logs if possible
I'm looking forward to your response.

Kind Regards,
Xander

**Comment by Rajkumar2432** [comment]
My device is in iPadOS 15 Beta 8
Log: when app tries to present usabilla form it fails and call "formDidFailLoading" with error "Unable to load the form"
Steps: Form is not loaded when tapping the respective button , it is working on iOS 14 and on iOS 15 when running from Xcode 13. I am facing this issue only on Xcode 12.5 build on iOS 15 device.

**Comment by Rajkumar2432** [comment]
I tried both v6.6.1 and v6.8.4of Usabilla ... both are having the same issue

**Comment by xanderwernecke** [comment]
Hi @Rajkumar2432 . Thank you for providing me with the additional details, I've passed this information onto the developers and I'll update you on this once I hear back from them.

Thank you for your patience.

Kind Regards,
Xander

**Comment by xanderwernecke** [comment]
Hi @Rajkumar2432. Please could you update to v6.8.5 and let me know if you're still experiencing the same issue.

Kind Regards,
Xander

**Comment by Rajkumar2432** [comment]
I am still experiencing the same issue

**Comment by xanderwernecke** [comment]
Hi @Rajkumar2432 ,

Our Developer tested this on MacMini for xcode 12.5.1 with iPadOS 15 on a simulator and it worked as expected. He mentioned used he used the following commands: 
```
pod 'Usabilla', :podspec => 'https://raw.githubusercontent.com/usabilla/usabilla-u4a-ios-swift-sdk/Xcode-11.7/Usabilla.podspec'
pod install --repo-update
```
Please could you implement this and let me know if this worked.

Kind Regards,
Xander

**Comment by Rajkumar2432** [comment]
Hi Xander, Thanks for the response.
We are still facing the issue. But on further investigation we found that Usabilla is not initialising when the device is connected to VPN (even in Xcode 13 Beta/ iOS 15)

            Usabilla.initialize(appID: AppSettings.sharedInstance.usabillaFeedbackAppID) {
                print("Initialize Success")  **// This call-back code is not executed when device is connected to VPN**
            } 
            
 When VPN is not connected everything is working as expected. Also when I launch the app once without VPN connection from next launch onwards it is working even with VPN connection is ON
 
 In Short: Usabilla is not initialising when device is connected to VPN

**Comment by xanderwernecke** [comment]
Hi @Rajkumar2432 .

One of the Developers mentioned that Usabilla might be blocked on the VPN so they recommended to please reach out to your IT to get a certificate for this so that Usabilla isn't blocked over a VPN. 

Kind Regards,
Xander
